src/main/resources/org/jenkinsci/plugins/authorizeproject/form/dropdownDescriptorSelector.jelly

```diff
@@ -66,7 +66,7 @@ THE SOFTWARE.
       <f:dropdownListBlock value="${loop.index}" title="${descriptor.displayName}"
                            selected="${current.descriptor==descriptor || (current==null and descriptor==attrs.default)}"
                            staplerClass="${descriptor.clazz.name}"
-                           lazy="${attrs.lazy?attrs.lazy+',':''}descriptor,it">
+                           lazy="${(attrs.capture != null and attrs.capture != '')?attrs.capture+',':''}descriptor,it">
```

Comment:
I tested the behavior of expressions in jelly and conclude that jelly isn't groovy:

| Expression                   | capture is undefined | capture='authorizeProjectContext' | capture='true' | capture='' |
|:-----------------------------|:---------------------|:----------------------------------|:---------------|:-----------|
| attrs.capture?'true':'false' | false                | false                             | true           | false      |
| attrs.capture?:'false'       | false                | authorizeProjectContext           | true           | false      |

Manage Jenkins > Script Console works completely as groovy:

| Expression           | value = null | value='authorizeProjectContext' | value='' |
|:---------------------|:-------------|:--------------------------------|:---------|
| value?'true':'false' | false        | true                            | false     |
| value?:'false'       | false        | authorizeProjectContext         | false     |